stage/LibUsbDotNet/DeviceNotify/DeviceNotifyEventArgs.cs

```diff
@@ -103,10 +77,6 @@ public object Object
         ///<returns>
         ///A <see cref="System.String"/> that represents the current <see cref="DeviceNotifyEventArgs"/>.
         ///</returns>
-        public override string ToString()
-        {
-            object[] o = {DeviceType, EventType, mObject.ToString()};
-            return string.Format("[DeviceType:{0}] [EventType:{1}] {2}", o);
-        }
+        public override string ToString() => $"[DeviceType:{DeviceType}] [EventType:{EventType}] {Object}";

```

Comment:
The `ToString()` call on `mObject` is probably unnecessary and omitting it will avoid a NullReferenceException being thrown if `Object` is null when `ToString()` is called.

My own subclass does not set `Object` to anything I noticed this exception being thrown when I tried to view a string representation of my device notification events in the debugger - this is probably the cause of those.